File: vidaplus_app/lib/models/paciente.dart
class Paciente {
  final String id;
  final String nome;
  final String? cpf;
  final String? telefone;

  Paciente({
    required this.id,
    required this.nome,
    this.cpf,
    this.telefone,
  });

  factory Paciente.fromJson(Map<String, dynamic> json) {
    return Paciente(
      id: json['id'],
      nome: json['nome'],
      cpf: json['cpf'],
      telefone: json['telefone'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'nome': nome,
      'cpf': cpf,
      'telefone': telefone,
    };
  }
}

File: vidaplus-backend-tcc/src/main/java/br/com/vidaplus/vidaplus_backend_tcc/entity/Paciente.java
package br.com.vidaplus.vidaplus_backend_tcc.entity;

import com.fasterxml.jackson.annotation.JsonBackReference;
import jakarta.persistence.*;
import lombok.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Paciente {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private String id;

    @OneToOne
    @JoinColumn(name = "user_id")
    @JsonBackReference
    private User user;

    private String nome;
    private String cpf;
    private String telefone;
}

